### ISSUE TITLE
metrics - add L1 gradient norm tracking

### ISSUE THREAD

**Issue by lessw2020** [pytorch/torchtitan#119]
to help monitor training stability.

**Comment by tianyu-l** [comment]
closing since it's been idle for a while
